### PR TITLE
labels: add subsystem labels for files in src/

### PR DESCRIPTION
**Pull request by mscdex**

### CODE REVIEW

**Comment by Fishrock123** [comment]
@mscdex they don't need to also add `c++`, that is handled by https://github.com/nodejs/github-bot/pull/110/files#diff-571193c4855ed04ea7c66486806af968R37

File: lib/node-labels.js [hunk 1]
```diff
@@ -3,6 +3,34 @@
...
+  [/^src\/node_crypto/, ['c++', 'crypto']],
+  [/^src\/(?:debug-|node_debug)/, ['c++', 'debugger']],
+  [/^src\/udp_/, ['c++', 'dgram']],
+  [/^src\/(?:fs_|node_file|node_stat_watcher)/, ['c++', 'fs']],
+  [/^src\/node_http_parser/, ['c++', 'http_parser']],
+  [/^src\/node_i18n/, ['c++', 'intl']],
+  [/^src\/uv\./, ['c++', 'libuv']],
+  [/^src\/(?:connect(?:ion)?|pipe|tcp)_/, ['c++', 'net']],
+  [/^src\/node_os/, ['c++', 'os']],
+  [/^src\/(?:node_main|signal_)/, ['c++', 'process']],
```
Comment (Fishrock123) [inline lib/node-labels.js]: node_main should probably be `lib / src`?

**Comment by mscdex** [comment]
@Fishrock123 It is needed because of the way the `subSystemLabelsMap` is used (first match wins). This is described in the comment preceding `subSystemLabelsMap`.

File: lib/node-labels.js (hunk 1 above)
Comment (mscdex) [inline lib/node-labels.js]: To me it has to do with startup, so I think it belongs under the "main" `process`.

What do other @nodejs/github-bot members think?

**Comment by Fishrock123** [comment]
Oh hmmm, ok.

File: lib/node-labels.js [hunk 2]
```diff
@@ -3,6 +3,34 @@
...
+  [/^src\/timer_/, ['c++', 'timers']],
+  [/^src\/(?:CNNICHashWhitelist|node_root_certs|tls_)/, ['c++', 'tls']],
+  [/^src\/tty_/, ['c++', 'tty']],
+  [/^src\/node_url/, ['c++', 'url']],
```
Comment (targos) [inline lib/node-labels.js]: can you please add `dont-land-on-v6.x` and `dont-land-on-v4.x` ?

**Comment by phillipj** [comment]
Very nice improvement! Having to add `c++` to them all is kinda sad, but I'm absolutely for getting this landed compared to refactoring that out ATM.

File: lib/node-labels.js (hunk 1 above)
Comment (phillipj) [inline lib/node-labels.js]: Not too opinionated about this file, tho @mscdex reasoning about it being `process` makes sense.

**Comment by phillipj** [comment]
I'm getting some style issues when running `npm test` locally:

```bash
standard: Use JavaScript Standard Style (http://standardjs.com) 
  /../github-bot/test/unit/node-labels.test.js:143:3: Expected { after 'if' condition.
  /../github-bot/test/unit/node-labels.test.js:144:22: Extra semicolon.
```

**Comment by mscdex** [comment]
@phillipj Are you referring to the explicit `c++` labelling in the code or that changes to those files are labeled as `c++` at all?

**Comment by mscdex** [comment]
@phillipj I'm not getting those linter errors when running `npm test` locally.

**Comment by mscdex** [comment]
Lint issues fixed.

**Comment by phillipj** [comment]
> @phillipj Are you referring to the explicit c++ labelling in the code or that changes to those files are labeled as c++ at all?

The former.

> @phillipj I'm not getting those linter errors when running npm test locally.

Hm, double strange when hearing you're struggling with broken tests locally (refs https://github.com/nodejs/github-bot/issues/109) which I'm not able to reproduce.

Just flushed my `node_modules`, set node version to 6.9.1 as you said you're running, and still seeing those style issues with standard 6.0.8 installed. Really not a fan of "works on my computer" issues :/

**Comment by phillipj** [comment]
> Lint issues fixed.

@mscdex sure? E.g. I'm still seeing the semicolon on [line 144](https://github.com/nodejs/github-bot/pull/110/commits/b37144a1d044d384ebeb59eae6d58ecd2dfa2c7f#diff-e6016f10ced0ba0ebe44149ccd45bd8eR144) which was one of the issues I included in the standard output previously.

**Comment by mscdex** [comment]
> @mscdex sure? E.g. I'm still seeing the semicolon on line 144 which was one of the issues I included in the standard output previously.

I forgot to add it when I force pushed. It should be up there now...

**Pull request merged**